src/audits/accessibility/aria-valid-attr.js

```diff
@@ -43,6 +43,15 @@ class ARIAValidAttr extends Audit {
   }
 
   /**
+   * @return {!Array<string>}
```

Comment:
The signal to noise ratio in the audit files is already pretty low.

Can you try condensing these a bit?

e.g.

#### before:
```js
  /**
   * @return {!Array<string>}
   */
  static requiredArtifacts() {
    return [
      'accessibility'
    ];
  }
```
#### after:
```js
  /**
   * @return {!Array<string>}
   */
  static requiredArtifacts() { return ['accessibility']; }
```

#### or even
```js
  static /** {!Array<string>} */ requiredArtifacts() { 
    return ['accessibility']; 
  }
```